packages/slate/src/models/node.js

```diff
@@ -2023,77 +2023,65 @@ function assertKey(arg) {
  * Memoize read methods.
  */
 
-memoize(
-  Node.prototype,
-  [
-    'getBlocksAsArray',
-    'getCharactersAsArray',
-    'getFirstText',
-    'getInlinesAsArray',
-    'getKeysAsArray',
-    'getLastText',
-    'getMarksAsArray',
-    'getText',
-    'getTextDirection',
-    'getTextsAsArray',
-    'isLeafBlock',
-    'isLeafInline',
-  ],
-  {
-    takesArguments: false,
-  }
-)
-
-memoize(
-  Node.prototype,
-  [
-    'areDescendantsSorted',
-    'getActiveMarksAtRangeAsArray',
-    'getAncestors',
-    'getBlocksAtRangeAsArray',
-    'getBlocksByTypeAsArray',
-    'getCharactersAtRangeAsArray',
-    'getChild',
-    'getClosestBlock',
-    'getClosestInline',
-    'getClosestVoid',
-    'getCommonAncestor',
-    'getDecorations',
-    'getDepth',
-    'getDescendant',
-    'getDescendantAtPath',
-    'getFragmentAtRange',
-    'getFurthestBlock',
-    'getFurthestInline',
-    'getFurthestAncestor',
-    'getFurthestOnlyChildAncestor',
-    'getInlinesAtRangeAsArray',
-    'getInlinesByTypeAsArray',
-    'getMarksAtRangeAsArray',
-    'getInsertMarksAtRangeAsArray',
-    'getMarksByTypeAsArray',
-    'getNextBlock',
-    'getNextSibling',
-    'getNextText',
-    'getNode',
-    'getNodeAtPath',
-    'getOffset',
-    'getOffsetAtRange',
-    'getParent',
-    'getPath',
-    'getPlaceholder',
-    'getPreviousBlock',
-    'getPreviousSibling',
-    'getPreviousText',
-    'getTextAtOffset',
-    'getTextsAtRangeAsArray',
-    'validate',
-    'getFirstInvalidDescendant',
-  ],
-  {
-    takesArguments: true,
-  }
-)
+memoize(Node.prototype, [
+  'getBlocksAsArray',
+  'getCharactersAsArray',
+  'getFirstText',
+  'getInlinesAsArray',
+  'getKeysAsArray',
+  'getLastText',
+  'getMarksAsArray',
+  'getText',
+  'getTextDirection',
+  'getTextsAsArray',
+  'isLeafBlock',
+  'isLeafInline',
+])
+
+memoize(Node.prototype, [
```

Comment:
Can you combine these into a single one now, alphabetized?

Reply:
Fixed~